packages/zero-pg/src/custom.ts

```diff
@@ -1,12 +1,15 @@
 import type {ReadonlyJSONValue} from '../../shared/src/json.ts';
 import type {Schema} from '../../zero-schema/src/builder/schema-builder.ts';
+import type {TableSchema} from '../../zero-schema/src/table-schema.ts';
 import type {
   SchemaCRUD,
   SchemaQuery,
+  TableCRUD,
   TransactionBase,
 } from '../../zql/src/mutate/custom.ts';
 import type {ConnectionProvider, DBTransaction} from './db.ts';
 import {PushProcessor, type PushHandler} from './web.ts';
+import {formatPg, sql} from '../../z2s/src/sql.ts';
```

Comment:
we build our sql with the `z2s` query builder rather than pulling in a full-fledged postgres library. The reason is that the user can bring their own postgres library. We're not opinionated about this.